CHANGES.md

```diff
@@ -1,5 +1,29 @@
 # Changes
 
+## stable-2.13.4
+
+This stable release fixes a few issues in the proxy and in the outbound policy
+API. Two new configuration options are also introduced to configure the
+outbound (and inbound) cache discovery idle period for proxies. The
+configuration is supported through annotations and through Helm values.
+
+* Control Plane
+  * Fixed an issue where an HTTPRoute backendRef's namespace would be ignored
+  * Fixed an issue where probe paths with query params would not be authorized
+    through the default route
```

Comment:
take it or leave it: perhaps we should use the same wording as in the edge changelog for these:
https://github.com/linkerd/linkerd2/blob/06a6f3be1dcabc38700f62d54487386fc2230987/CHANGES.md?plain=1#L7-L11

```suggestion
  * Fixed an issue where the `namespace` field on HTTPRoute `backendRef`s was
    ignored, and the backend Service would always be assumed to be in the
    namespace as the parent Service
  * Fixed an issue where default authorizations generated for readiness and
    liveness probes would fail if the probe path included URI query parameters
```

Reply:
Sure, sounds like a good idea.